src/CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ if (WIN32)
                        ${DLL_LOCATION} ${UNIT_TEST_DIR})
 else(WIN32)
     find_package(LibXml2 2.9 REQUIRED)
-    find_package(CURL 7.31 REQUIRED)
+    find_package(CURL 7.29 REQUIRED)
```

Comment:
I'm not in position to backport and support newer version of CURL into the RHEL7/CentOS7. I would prefer to keep using the version provided and supported by RedHat until there're requirements to use newer version of CURL.

So that I will revert change of src/CMakeFile.txt and fix it from the %build step